chromium_src/chrome/browser/permissions/permission_manager_factory.cc

```diff
@@ -32,6 +38,12 @@ KeyedService* PermissionManagerFactory::BuildServiceInstanceFor(
       permission_context.second->SetPermissionLifetimeManagerFactory(factory);
     }
   }
-  return new permissions::PermissionManager(profile,
-                                            std::move(permission_contexts));
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+  permission_contexts[ContentSettingsType::BRAVE_ETHEREUM] =
```

Comment:
this should be moved up, right after ```CreatePermissionContexts```.

Reply:
Fixed in https://github.com/brave/brave-core/pull/9443/commits/b127eefe84bac3157ee00dbdfef764ebf6a691a6